src/main/java/com/minecolonies/coremod/entity/ai/citizen/miner/EntityAIStructureMiner.java

```diff
@@ -927,4 +946,60 @@ protected void onStartWithoutStructure()
          * Nothing to do here.
          */
     }
+
+    @Nullable
+    private BlockPos findFirstLevelSign(final StructureWrapper structure)
+    {
+        for (int j = 0; j < structure.getHeight(); j++)
```

Comment:
Can't we get this info during the construction of the node?
Similar to the register to building code of the builder?
So we avoid this search here?

Reply:
-> Discord